### PR TITLE
Testing Your Ruby Code - Tic Tac Toe

### PR DESCRIPTION
**Pull request by camfilho**
# Hello
This is the part where we test our Tic Tac Toe Project with RSpec.

### CODE REVIEW

File: spec/tictactoe_spec.rb [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        subject.move('o', 4)
+        subject.move('x', 3)
+        expect(subject.board).to eql [1, 2, 'x', 'o', 5, 6, 7, 8, 9]
+      end
+    end
+    context 'When not empty' do
+      subject { described_class.new }
+      it 'raise an exception if position taken' do
+        subject.move('o', 4)
+        expect { subject.move('x', 4) }.to raise_exception('Wrong coordinates')
```
Comment (Redvanisation) [inline spec/tictactoe_spec.rb]: This spec is failing, change the line `89` to the following instead: `expect { subject.move('x', 4) }.to raise_exception RuntimeError`

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**You have done an amazing job on this project, however, we need to implement the following changes before moving on ⬇️** 

1. You have tested all the public methods in the project, that's great! we need to test for both **positive** and **negative** cases, you already did the first one, and only the implementation of the negative case scenarios tests is missing for all the methods.

2. There is a failing test related to the `move` method, please check my comment below on how to fix it.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by Ispirett — APPROVED** [review]
## status approved :+1:  :100:  :1st_place_medal: 
great  work @Redvanisation  @mkezmkez 

#### One point
It is good practice to not log any display msg with test.
Currently,  its displaying the board.

```
 x  |  x  |  3
----|-----|-----
 x  |  5  |  6
----|-----|-----
 7  |  8  |  9
 
 
 x  |  x  |  3
----|-----|-----
 x  |  x  |  6
----|-----|-----
 7  |  8  |  9
 
      not equal true if not met winner requirements
 
 x  |  2  |  3
----|-----|-----
 4  |  5  |  6
----|-----|-----
 7  |  8  |  9
 
 
 x  |  x  |  3
----|-----|-----
 4  |  5  |  6
----|-----|-----
 7  |  8  |  9
 
 
 x  |  x  |  x
----|-----|-----
 4  |  5  |  6
----|-----|-----
 7  |  8  |  9
 
      returns true if x or o wins
  #swtich player
    when the turn is to the next player
      switches player
      is equal to the next player



```
